peakorc.py

```diff
@@ -172,11 +172,14 @@ def on_get(self, req, resp, suite_uuid):
 peak_test = PeakTestResource()
 peak_suite_tests = PeakSuitesTestsDetailResource()
 
-api.add_route('/test_suite/new', peak_suite_new)
+# suites
 api.add_route('/suites/', peak_suites)
+api.add_route('/suites/new', peak_suite_new)
```

Comment:
I think we can omit this particular route. Instead, a POST request to /suites/ would create a new test suite resource and return the resources as the response to the user.

Reply:
Completely agree, it's definitely more REST-like, and the other route was redundant. I've just updated this branch and also https://github.com/peak-oss/peakweb/pull/8, as the old route was referenced in `peakweb`. Thanks!